fuzzers/fuzz1.cpp

```diff
@@ -0,0 +1,27 @@
+#include <string>
```

Comment:
Can we rename this to indicate what's being fuzzed? For example, "fuzz_cjson_to_string.cpp" or similar.